assisted_deployment.sh

```diff
@@ -273,26 +281,38 @@ function patch_extra_host_manifests() {
 }
 
 function install_assisted_service() {
- mkdir -p "${ASSETS_DIR}"
- patch_extra_host_manifests
- deploy_local_storage
- deploy_hive
- deploy_assisted_operator
+  install_prerequisites_assisted_service
+  deploy_assisted_operator
 
- oc wait -n "$ASSISTED_NAMESPACE" --for=condition=Ready pod -l app=assisted-service --timeout=90s
+  oc wait -n "$ASSISTED_NAMESPACE" --for=condition=Ready pod -l app=assisted-service --timeout=90s
+
+  echo "Installation finished..."
+  echo "For debugging purposes all the manifests have been saved in ${OCP_DIR}/saved-assets/assisted-installer-manifests"
+  echo "Please remember to manually apply BareMetalHost manifest available in the directory above."
```

Comment:
It will be handy to just also print the command `oc apply -f ${OCP_DIR}/saved-assets/assisted-installer-manifests/bmh-manifest-filename`

Reply:
Also, can we not execute the oc apply from the script?

Reply:
Done